File: samples_generated_code/samples_1/qwen/baseline_code_java/code_row_64.java
import java.util.Scanner;

public class ArrayIndexValue {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int[] id_sequence = {10, 20, 30, 40, 50};

        System.out.print("Enter an index value: ");
        int index = scanner.nextInt();

        if (index >= 0 && index < id_sequence.length) {
            int value = getValueAtIndex(id_sequence, index);
            System.out.println("The value at index " + index + " is: " + value);
        } else {
            System.out.println("Index out of bounds.");
        }

        scanner.close();
    }

    public static int getValueAtIndex(int[] array, int index) {
        return array[index];
    }
}

File: samples_generated_code/sample_1/athene/baseline_code_py/code_row_64.py
def get_value_by_index(index):
    id_sequence = [10, 20, 30, 40, 50]
    return id_sequence[index]

index = int(input("Enter an index: "))
try:
    print(get_value_by_index(index))
except IndexError:
    print("Index out of range")